Resources/Prototypes/Entities/Objects/Specific/Janitorial/janitor.yml

```diff
@@ -203,3 +203,30 @@
           maxFillLevels: 3
           fillBaseName: cart_water_
           changeColor: false
+
+
+- type: entity
+  id: FloorDrain
+  name: drain
+  description: Drains puddles around it. Useful for dumping mop buckets or keeping certain rooms clean.
+  placement:
+    mode: SnapgridCenter
+  components:
+    - type: Sprite
+      netsync: false
+      drawdepth: FloorObjects
+      sprite: Objects/Specific/Janitorial/drain.rsi
+      state: icon
+    - type: Physics
+      bodyType: Static
+    - type: Drain
+    - type: AmbientSound
+      enabled: false
+      volume: -10
```

Comment:
not sure how it works in game but i'd be fine with it being a little loud since its a drain